README.md

```diff
@@ -32,20 +855,1218 @@ The written tests available in the directory [`test`](./test) achieve a test cov
 | src/CreateX.sol | 100.00% (149/149) | 100.00% (210/210) | 100.00% (78/78) | 100.00% (31/31) |
 ```
 
-> **Important:** A test coverage of 100% does not mean that there are no vulnerabilities. What really counts is the quality and spectrum of the tests themselves!
+> **Note**<br>
+> A test coverage of 100% does not mean that there are no vulnerabilities. What really counts is the quality and spectrum of the tests themselves!
 
-## Security Considerations
+## ABI (Application Binary Interface)
+
+<details>
+<summary> <a href="https://docs.soliditylang.org/en/latest/">Solidity</a> </summary>
```

Comment:
Should we instead link to the `ICreateX.sol` file to make that more discoverable, and users can then `forge install` this repo?

Reply:
I think we can do both. Keeping it in the `README` (I like the easy copy+paste feature from GH for these cases) and add a link. Added a note here with the link: https://github.com/pcaversaccio/createx/pull/26/commits/9ee14ae3bfd6322602d8adac01602c6fa20febfc. Also, it's already on the top of the `README` 
![image](https://github.com/pcaversaccio/createx/assets/25297591/8d57fe65-0578-4fb1-9773-1bd2fdcbc65a)